spec/features/signing_in_and_out_spec.rb

```diff
@@ -0,0 +1,20 @@
+
+feature 'User sign in' do
+
+  scenario 'with correct credentials' do
+     user = create :user
+     sign_in(user)
+     expect(page).to have_content "Welcome, #{user.email}"
+   end
```

Comment:
Lint/BlockAlignment: `end` at 8, 3 is not aligned with `scenario 'with correct credentials' do` at 4, 2